src/main/java/se/mickelus/tetra/module/data/TweakData.java

```diff
@@ -53,4 +56,12 @@ public float getCapabilityEfficiency(Capability capability, int step) {
     public int getMagicCapacity(int step) {
         return baseStats.magicCapacity + step * stepStats.magicCapacity;
     }
+
+    public Map<ResourceLocation, Object> getBaseStatsCustom() {
+        return baseStats.custom;
+    }
+
+    public Map<ResourceLocation, Object> getStepStatsCustom() {
+        return stepStats.custom;
+    }
```

Comment:
Since the stats are both defined as `ModuleVariantData`, this exposes the custom field of them, should someone wish to make use of it in the context of tweaks. A better alternative may be to make the fields themselves public.